docs_src/microservices/application/V3Migration.md

```diff
@@ -1,336 +1,231 @@
 # V3 Migration Guide
 
-!!! warning
-    Updates to this migration guide for V3 are still pending. Content/structure below is from V2
+## Configuration
 
-## Custom Application Services
+The migration of any Application Service's configuration starts with migrating configuration common to all EdgeX services. See the [V3 Migration of Common Configuration](../../configuration/V3MigrationCommonConfig) section for details including the change from **TOML** format to **YAML** format for the configuration file. The remainder of this section focuses on configuration specific to Application Services.
 
-### Configuration
+### Common Configuration Removed
 
-The migration of any Application Service's configuration starts with migrating configuration common to all EdgeX services. See the [V2 Migration of Common Configuration](../../configuration/V2MigrationCommonConfig) section for details. The remainder of this section focuses on configuration specific to Application Services.
+All configuration that is common to all EdgeX services or all EdgeX Application Services needs to be removed from custom application service's private configuration. See [Common Service Configuration](../../configuration/CommonConfiguration/) section for details about configuration that is common to all Edgex services. See [Application Service Configuration](../GeneralAppServiceConfig) section for details about configuration that is common to all EdgeX Application Services.
 
-#### SecretStoreExclusive
+!!! note
+    With this change, the your custom application service must be run with either the `-cp--configProvider` flag or the `-cc/-commonConfig` flag in order for the service to receive the common configuration that has been removed from its private configuration. See [Config Provider](../../configuration/CommonCommandLineOptions/#config-provider) and [Common Config](../../configuration/CommonCommandLineOptions/#common-config) sections for more details on these flags.
```

Comment:
should it be `--commonConfig` or `-commonConfig` ?

Reply:
also, I think `-cp--configProvider` should be `-cp/--configProvider`

Reply:
fixed